src/gui/optionsdlg.cpp

```diff
@@ -344,9 +344,20 @@ OptionsDialog::OptionsDialog(QWidget *parent)
     connect(m_ui->spinMaxActiveDownloads, qSpinBoxValueChanged, this, &ThisType::enableApplyButton);
     connect(m_ui->spinMaxActiveUploads, qSpinBoxValueChanged, this, &ThisType::enableApplyButton);
     connect(m_ui->spinMaxActiveTorrents, qSpinBoxValueChanged, this, &ThisType::enableApplyButton);
-    connect(m_ui->checkIgnoreSlowTorrentsForQueueing, &QAbstractButton::toggled, this, &ThisType::enableApplyButton);
+    connect(m_ui->checkIgnoreSlowTorrentsForQueueing, &QGroupBox::toggled, this, &ThisType::enableApplyButton);
+    connect(m_ui->spinDownloadRateForSlowTorrents, qSpinBoxValueChanged, this, &ThisType::enableApplyButton);
+    connect(m_ui->spinUploadRateForSlowTorrents, qSpinBoxValueChanged, this, &ThisType::enableApplyButton);
+    connect(m_ui->spinSlowTorrentsInactivityTimer, qSpinBoxValueChanged, this, &ThisType::enableApplyButton);
     connect(m_ui->checkEnableAddTrackers, &QGroupBox::toggled, this, &ThisType::enableApplyButton);
     connect(m_ui->textTrackers, &QPlainTextEdit::textChanged, this, &ThisType::enableApplyButton);
+
+    const QString slowTorrentsExplanation = QLatin1String("<html><body><p>")
```

Comment:
I would have this text in .ui file (as tooltip for one of labels) and duplicate it here, e.g.:
```c++
m_ui->labelUploadRateForSlowTorrents->setToolTip(m_ui->labelDownloadRateForSlowTorrents->tooltip());
m_ui->labelSlowTorrentInactivityTimer->setToolTip(m_ui->labelDownloadRateForSlowTorrents->tooltip());
```

Reply:
Are you sure? I don't see any other html in the .ui but in the .cpp [there is this](https://github.com/qbittorrent/qBittorrent/blob/master/src/gui/optionsdlg.cpp#L121). Maybe to avoid including the tags in the translation?

Reply:
>Are you sure?

No, of course.
>Maybe to avoid including the tags in the translation?

Reasonable.

Reply:
I take that i should leave it as is then.

Reply:
I have no more comments.